src/molecules/RegistrationForm/index.js

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import {useState} from "react";
+import './style.scss';
+import Typography from "../../atoms/Typography";
+import Button from "../../atoms/Button";
+import {Link} from "react-router-dom";
+import axios from 'axios';
+
+function RegistrationForm() {
+    const [formData, setFormData] = useState({
+        name: '',
+        email: '',
+        password: '',
+    });
+
+    const [response, setResponse] = useState(null);
+    const [error, setError] = useState(null);
+
+    const handleChange = (event) => {
+        const { name, value } = event.target;
+        setFormData({
+            ...formData,
+            [name]: value,
+        });
+    }
+
+    const handleSubmit = (event) => {
+        event.preventDefault();
+
+        axios.post('/api/v1/auth/register', formData)
+            .then((response) => {
+                setResponse({
+                    username: response.data.username,
+                    jwtToken: response.data.jwtToken,
+                });
+                setError(null);
+                console.log("Response from the server:");
```

Comment:
We can have those logs on development stage as example for debugging, but I don't think it will be a good idea to have them in main code base.